### PR TITLE
Create documentation using mdoc and Docusaurus

### PR DESCRIPTION
**Pull request by robinraju**

### CODE REVIEW

File: docs/reference-studentify.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  -dot, --dotty            studentified repository is a Dotty project
+  -nar, --no-auto-reload-sbt
+                           no automatic reload on build definition change
+  --help                   Prints the usage text
+  -v, --version            Prints the version info
+```
+
+# Utilisation
+
+![studentify process](https://i.imgur.com/RFTpBYR.png)
```
Comment (robinraju) [inline docs/reference-studentify.md:33]: The image link is broken

Comment (eloots) [reply docs/reference-studentify.md:33]: Indeed, that's because I was making a few changes to the images and I need to update the image name

Comment (eloots) [reply docs/reference-studentify.md:33]: > Looks like these files got added accidentally

These are source files in Excel & Omnigraffle which were modified, so ok.

**Comment by eloots** [comment]
> Since I created this PR with an initial commit, Github is preventing me from approve it 😆

Then I'll review my own changes!

😂

**Pull request merged**